### PR TITLE
add comments

### PR DESCRIPTION
**Pull request by tinapgaara**

### CODE REVIEW

**Comment by tinapgaara** [comment]
Add comments.

File: src/folding_planner/src/FoldPlanner.cpp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 FoldPlanner::FoldPlanner()
 {
-
+ // Constructor.
```
Comment (Yinxiaoli) [inline src/folding_planner/src/FoldPlanner.cpp]: Move outside.

Comment (Yinxiaoli) [inline src/folding_planner/src/FoldPlanner.cpp]: Move outside.

**Review by Yinxiaoli — CHANGES_REQUESTED** [review]
Please address the change.

**Review by tinapgaara — COMMENTED** [review]
Move the comment outside the func.

**Review by Yinxiaoli — APPROVED** [review]

**Pull request merged**